plotly/plotly/plotly.py

```diff
@@ -212,7 +297,8 @@ def iplot_mpl(fig, resize=True, strip_style=False, update=None, **plot_options):
             fig.update(update)
             fig.validate()
         except exceptions.PlotlyGraphObjectError as err:
-            err.add_note("Your updated figure could not be properly validated.")
+            err.add_note("Your updated figure could not be properly " +
+                         "validated.")
```

Comment:
:cow2: Python's smart enough to concatenate your strings for you:

```
err.add_note("Your updated figure could not be properly"
             "validated")
```

This works because of the `()`